test/integration/master/kms_transformation_test.go

```diff
@@ -139,13 +141,19 @@ func TestKMSProvider(t *testing.T) {
 	}
 }
 
-func getDEKFromKMSPlugin(pluginMock *base64Plugin) ([]byte, error) {
-	select {
-	case e := <-pluginMock.encryptRequest:
-		return e.Plain, nil
-	case <-time.After(time.Second):
-		return nil, fmt.Errorf("timed-out while getting encryption request from KMS Plugin Mock")
+func getDEKFromKMSPlugin(pluginMock *base64Plugin, serveErrorChan chan error) ([]byte, error) {
+	if len(serveErrorChan) != 0 {
```

Comment:
It seems like you should check this outside the function

Reply:
Yes, makes perfect sense.